### PR TITLE
feat(profile): added login screen

### PR DESCRIPTION
**Pull request by Vojtech-Sassmann**
* Added support for login screen to the user profile.

### CODE REVIEW

**Comment by Vojtech-Sassmann** [comment]
Unfortunately, I didn't manage to use the shared base layout component, because the profile is using its custom footer and the base layout is a bit different than in the other applications. In the future, it would be good to unify the footer and use one footer for all of our apps.

File: apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="app-min-width">
+  <perun-web-apps-header [hideToggle]="true"></perun-web-apps-header>
+  <div [ngStyle]="{'min-height': getContentHeight()}" id="content-wrap">
+    <perun-web-apps-breadcrumbs></perun-web-apps-breadcrumbs>
```
Comment (xkureck) [inline apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html:4]: I dont think it needs to be here

File: apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="app-min-width">
+  <perun-web-apps-header [hideToggle]="true"></perun-web-apps-header>
```
Comment (xkureck) [inline apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html:2]: The logo in this component is clickable

**Review by xkureck — CHANGES_REQUESTED** [review]

File: apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html (hunk 1 above)
Comment (Vojtech-Sassmann) [reply apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html]: You are right, I have removed it.

File: apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html (hunk 2 above)
Comment (Vojtech-Sassmann) [reply apps/user-profile/src/app/components/profile-login-screen-base/profile-login-screen-base.component.html]: Fixed.

**Review by xkureck — APPROVED** [review]

**Pull request merged**